tests/functional/testOnboarding.js

```diff
@@ -12,24 +12,23 @@ describe('Onboarding', function() {
 
   beforeEach(async () => {
     await vpn.flipFeatureOn("newOnboarding");
-    assert.equal(await vpn.getSetting('onboardingStarted'), false);
     assert.equal(await vpn.getSetting('onboardingCompleted'), false);
+    assert.equal(await vpn.getSetting('onboardingStep'), 0);
     await vpn.authenticateInApp();
   });
 
   //Tests pretty much all of the new onboarding, including all slides, controls, navigation, and ensuring settings persist into StateMain
   it('Complete onboarding', async () => {
     await vpn.waitForQuery(queries.screenOnboarding.SCREEN.visible());
     await vpn.waitForQuery(queries.screenOnboarding.ONBOARDING_VIEW.visible());
-    assert.equal(await vpn.getSetting('onboardingStarted'), true);
 
     //DATA SLIDE
     //Do some checks on the initial state of the data slide
     await vpn.waitForQuery(queries.screenOnboarding.DATA_SLIDE.visible());
     assert.equal(await vpn.getQueryProperty(queries.screenOnboarding.ONBOARDING_VIEW, 'currentIndex'), 0);
     assert.equal(await vpn.getSetting('onboardingStep'), 0);
     assert.equal(await vpn.getQueryProperty(queries.screenOnboarding.DATA_CHECKBOX, 'isChecked'), 'false');
-    assert.equal(await vpn.getSetting('gleanEnabled'), false);
+    assert.equal(await vpn.getSetting('gleanEnabled'), true);
```

Comment:
So right off the bat, we are collecting telemetry and then once the user gets to the consent page on Onboarding they can opt out if they choose to?

Reply:
yep. their selection will take affect once onboarding is completed